static/js/search.js

```diff
@@ -1,176 +1,119 @@
-var lunrIndex, pagesIndex;
-
-function endsWith(str, suffix) {
-    return str.indexOf(suffix, str.length - suffix.length) !== -1;
+// const SEARCH_API_URL = "https://docsearch-dot-redislabs-university.appspot.com/search"
+const SEARCH_API_URL = "https://docsearch-dot-redislabs-university.appspot.com/search"
```

Comment:
Once we have a load balancer set up, we need to point this app at the URL that we load-balance across regions instead of a specific App Engine instance, like we do here.